forge-gui/res/cardsfolder/upcoming/dawn_of_a_new_age.txt

```diff
@@ -0,0 +1,14 @@
+Name:Dawn of a New Age
+ManaCost:1 W
+Types:Enchantment
+K:etbCounter:HOPE:X
```

Comment:
missing Description

Reply:
Please test your changes...it's annoying:
![image](https://github.com/Card-Forge/forge/assets/8506892/a0975f5e-4d3a-4839-a1da-7f1f80342aaa)